### PR TITLE
add unit test case of account and common

### PR DESCRIPTION
**Pull request by qiluge**
the compact_uint_test test error, because of  the index out of range. See the $GOROOT/src/encoding/binary/binary.go, line 76 (bounds check hint to compiler; see golang.org/issue/14808).

### CODE REVIEW

File: account/account_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	accounts := make([]*Account, len(names)+1)
+	accounts[0] = NewAccount("")
+	assert.NotNil(t, accounts[0])
+	assert.NotNil(t, accounts[0].PrivateKey)
+	assert.NotNil(t, accounts[0].PublicKey)
+	assert.NotNil(t, accounts[0].Address)
+	assert.NotNil(t, accounts[0].PrivKey())
+	assert.NotNil(t, accounts[0].PubKey())
+	assert.NotNil(t, accounts[0].Scheme())
+	assert.Equal(t, accounts[0].Address, types.AddressFromPubKey(accounts[0].PublicKey))
```
Comment (laizy) [inline account/account_test.go]: you can add `""` in names to avoid handle  this special case .

File: account/file_store_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	walletReadFromFile.Load(WALLET_FILENAME)
+	assert.Equal(t, walletReadFromFile, wallet)
+	wallet.DelAccount(2)
+	assert.Equal(t, 1,len(wallet.Accounts))
+	assert.Panics(t, func(){wallet.DelAccount(2)})
+	wallet.Save(WALLET_FILENAME)
+	walletReadFromFile.Load(WALLET_FILENAME)
+	assert.Equal(t, walletReadFromFile, wallet)
+	deaultAccount := wallet.GetDefaultAccount()
+	assert.Equal(t, deaultAccount, accout)
```
Comment (laizy) [inline account/file_store_test.go]: delete the `wallet.dat` file before function return

File: common/common_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+)
+
+func TestHexAndBytesTransfer(t *testing.T) {
+	testBytes := []byte("10, 11, 12, 13, 14, 15, 16, 17, 18, 19")
+	stringAfterTrans := ToHexString(testBytes)
+	fmt.Println(stringAfterTrans)
```
Comment (laizy) [inline common/common_test.go]: do not print in test file

File: common/common_test.go [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	bytesAfterTrans, err := HexToBytes(stringAfterTrans)
+	fmt.Println(string(bytesAfterTrans))
+	assert.Nil(t, err)
+	assert.Equal(t, testBytes, bytesAfterTrans)
+}
+
+func TestGetNonce(t *testing.T) {
+	nonce1 := GetNonce()
+	fmt.Println(nonce1)
```
Comment (laizy) [inline common/common_test.go]: and here

**Review by laizy — CHANGES_REQUESTED** [review]

**Comment by qiluge** [comment]
Thanks，have modified。

**Review by laizy — APPROVED** [review]

**Pull request merged**